docs/data/date-pickers/date-picker/CustomInput.js

```diff
@@ -54,7 +54,7 @@ export default function CustomInput() {
         label="Custom input"
         defaultValue={dayjs('2022-04-07')}
         components={{
-          Input: BrowserInput,
+          TextField: BrowserInput,
```

Comment:
Thinking about it with a fresh head. I think I've come up with another reasonable naming option—`FieldInput`.
Pros over `TextField`:
- not directly related to the actual component used for the slot
- technically the field might not be text if the date format does not include date sections with words
- it's closed to what the actual slot replaces—a elaborate input component

Reply:
It was also my first idea before yesterday's meeting.
But I would prefer avoiding the term `Input` altogether :+1: